<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageProjectWindow</class>
 <widget class="QDialog" name="ManageProjectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Project</string>
  </property>
  <widget class="QPushButton" name="createBtn">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>320</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Project description:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="projectDescriptionLE">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>581</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>251</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Project name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="projectNameLE"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>254</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Max team size:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="maxTeamSizeLE"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>251</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Min team size:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="minTeamSizeLE"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>320</y>
     <width>102</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>361</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>createBtn</zorder>
  <zorder>label_6</zorder>
  <zorder>projectDescriptionLE</zorder>
  <zorder>cancelButton</zorder>
  <zorder>statusLbl</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
